typescript/web/src/components/workspaces/update-plan-banner.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, chakra, Flex, Text } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import { RiInformationFill } from "react-icons/ri";
+import { useWorkspace } from "../../hooks/use-user";
+
+const InfoIcon = chakra(RiInformationFill);
+export const UpdatePlanBanner = () => {
+  const { slug: workspaceSlug } = useWorkspace();
+  const router = useRouter();
+  const redirectToSettings = () => {
+    router.push(`/${workspaceSlug}/settings`);
+  };
+  return (
+    <Flex
+      direction={{ base: "column", md: "row" }}
+      justify="center"
+      align="center"
+      bg="#EFAB22"
+      zIndex="100"
```

Comment:
You shouldn't have to use zIndex but place the banner at the correct place in the DOM tree